website/data/tools/rollup/usage.md

```diff
@@ -1,6 +1,6 @@
 ```js
 var rollup = require("rollup");
```

Comment:
Not related to this PR, but can you also update the recommended usage according to https://github.com/rollup/plugins/tree/master/packages/babel#usage?

Reply:
Sure, I'll do that!